install/cluster-operator/020-RoleBinding-strimzi-cluster-operator.yaml-e

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Wrongly committed file? There seems to be a bunch of them.

Reply:
The build produced a bunch of these inside the source tree - I assumed these should be added like most of the other autogenerated things inside the source tree. So these yaml-e files _should_ be generated as they currently are, but _should not_ be included in git? What are they for?

Reply:
They should not have the `.yaml-e` extension. It should be just `.yaml`. This has to be something in your environment causing this.

Reply:
No idea how they got generated, can't reproduce it.